### DISCUSSION TITLE
course finalization for fall 2021 semester

### DISCUSSION THREAD

**Discussion by systemnerd** [snigi-gupta/AttendingUniversityAtBuffalo#55] [Q&A] [answered]
Hello,
I have decided to enroll for the following courses:

1. Algorithms by Roger HE
2. Intro to ML by Mingchen
3. Introduction to parallel and distributed programming by Zola
4. Operating systems by Farshad

Can you please evaluate on how balanced this coursework is? I believe this is on the tougher side

**Comment by snigi-gupta** [comment, marked as answer]
@NithinSastry 
It seems on the hard study load side since you have 2 hard courses-> PDP and OS.
Probably you can keep 1 hard and do the other one next sem.

You can look through my cse gists and balance them out. I am not sure how familiar you are already with the course concepts, so hard to say how much hard it will be for you.

For example, if I would have been in your place, I would have at max taken 1 hard course. But then that is me. My learning pace is different from yours. asses yourself is what I would say.

**Reply by EvaPradhan1** [reply]
Hello
I have the same course combination right now. Will it be any better if I replace os by computer vision image processing?
